Readme Documentation Updates

A-Camp opened the pull request:
Issue: https://github.com/codeforboston/voiceapp311/issues/204

Separate WIKI updates have been made. 

This PR just updates the README to outline existing Intents and how to use them for more casual users visiting. 

<img width="886" alt="Screen Shot 2019-03-19 at 9 21 00 PM" src="https://user-images.githubusercontent.com/4612974/54652352-ff437a80-4a8c-11e9-9d53-c7f99213e553.png">
<img width="895" alt="Screen Shot 2019-03-19 at 9 21 09 PM" src="https://user-images.githubusercontent.com/4612974/54652353-ff437a80-4a8c-11e9-86c2-5fbd025064ee.png">
<img width="891" alt="Screen Shot 2019-03-19 at 9 21 16 PM" src="https://user-images.githubusercontent.com/4612974/54652354-ff437a80-4a8c-11e9-896c-c59860612c25.png">
<img width="890" alt="Screen Shot 2019-03-19 at 9 21 23 PM" src="https://user-images.githubusercontent.com/4612974/54652355-ff437a80-4a8c-11e9-958b-538aaec520ce.png">

thiteixeira commented:
I like it! A couple of comments:
I'd move the developement paragraph to the end and add more intents under development or proposed (farmers markets, grocery stores, moving trucks, ...). You can filter the issues by the `new-intent` tag to get a list of proposed/under development intents

A-Camp commented:
Updated the Intents in Progress list and moved the Developer text to the bottom underneath a "Want to Help?" header

<img width="813" alt="Screen Shot 2019-03-26 at 7 48 06 PM" src="https://user-images.githubusercontent.com/4612974/55040745-2b687980-5000-11e9-940d-8a8db0594a06.png">

jmartini commented on README.md:
```diff
@@ -2,6 +2,105 @@
+
+
+#### Find any current Alerts
+This is used to getting any alerts for the day put out by the Boston government. The alerts can be about any of the the following services:
```
Typo: "used to getting" could be "used to get" or "used for getting"

jmartini commented on README.md:
```diff
@@ -2,6 +2,105 @@
+
+
+### Find any nearby Nearby Food Trucks
+The gets near-by food trucks for the given or stored Address. It returns the first five food trucks it finds within one mile of the given address as well as the time period that the food truck is expected to be at the location.
```
Typo: "The gets..."

jmartini commented on README.md:
```diff
@@ -2,6 +2,105 @@
+- Information about Moving permits
+- Adding yard waste pick-up days to our existing Trash Info Intent
+- An intent that combines the information of our most popular intents into one question/response pattern
+- A separate Brookline Info skill
```
I wouldn't list these here since this could easily get out of date as we abandon intents. Is it possible to link to the issue page filtered with the new intent tag?

A-Camp commented:
Food Truck description added - should be okay for final review/approve

A-Camp commented:
@jmartini I agree with the reasoning behind all your changes, latest PR version should have all of them. Thanks for the review!
